README.md

```diff
@@ -23,6 +23,32 @@ is not required to simply extract images.
 documentation. You can use `brew` if you have a Mac, but the
 steps are slightly more complicated for Windows or Unix users.
 
+## Running with Docker
+
+### Requirements
+The only requirement is Docker.
+
+### Build Docker Image
+```
+docker build -t extract_images_from_pdf .
+```
+### Docker Run
+
+Example Usage. Upon completion, the container will be removed.
+
+```
+docker run --rm \
```

Comment:
Your example should show where to put the argument. I attempted to run as is:
```bash
docker run --rm -v $(pwd):/output extract_images_from_pdf:latest bash
```
and got an assertion error on line 29 `assert os.path.exists(file_name)`, since (obv) I did not specify the file name upon run time. This should be made clear.